Library/HelpType.swift

```diff
@@ -23,6 +24,8 @@ public enum HelpType: SettingsCellTypeProtocol, CaseIterable {
     switch self {
     case .helpCenter:
       return Strings.Help_center()
+    case .community:
+      return ""
```

Comment:
TBD?

Reply:
So we're doing the same thing for the `trust` enum case here. Not sure If I should leave this blank as well or submit a new string like "Community Guidelines" for translation?

Reply:
Oh I guess these titles are for when we used to show the activity sheet, I don't think we do anymore so we can leave this for now.